README.md

```diff
@@ -1 +1,168 @@
-# Ruby-Group-Capstone
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Index
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Author](#author)
+- [🔭 Features](#features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of Things] `[Ruby, Rspec]` <a name="about-project"></a>
+
+
+This project is basically a console application designed for users to maintain a record of their personal items such as books, music albums, and games. The entire system designed on basis of UML diagram. The data stored in JSON files and a SQL database model is used to enhance its scalability.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+    <summary>Client</summary>
+    <ul>
+      <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    </ul>
+  </details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://www.json.org/json-en.html">Data Storaged In Json Files</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+  - **[Introduce additional books into the collection]
+  - **[Include new music albums to expand the library's selection]
+  - **[Incorporate new games into the gaming catalog]
+  - **[Maintain a record of authors, labels, and genres for future reference]
+  - **[Ensure data preservation by storing it in JSON files.]
+  - **[Conduct RSpec unit tests to verify functionality of code.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ To have a computer, Internet, Keyboard and Mouse, also you need to have Ruby settle up on your system
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  Open it with Visual Studio Code (or your preffered IDE), and open a server with "LiveServer".
+```
+
+### Install
+
+To run this project you need to install the following gems:
+
+```sh
+Colorize - Gem install Colorize
+Rubocop - Gem install Rubocop
+```
+
+### Usage
+
+To run this project, execute the following command:
+
+```sh
+Ruby main.rb
+```
+
+### Deployment
+
+This is only the Back-End part of the project, so It doesn't need any deploy
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
```

Comment:
- Kindly record a video presenting the features of the project with a maximum length of the video is 5 minutes which contains a clear and objective description of the project to fit the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md) 👍